index.md

```diff
@@ -8,6 +8,469 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+### C0102 Blacklisted name {#C0102}
+
+This error occurs when a variable name is listed in the black list. The black 
+list includes names like foo, bar, baz, toto, tata, tutu and so on. Students 
+are expected to use good variable names such as i, j, k, ex, Run. The example 
+below uses blacklisted names.
+
+```python 
+def pos(obj):
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    foo = obj
+    if foo < 0:
+        return False
+    else:
+        return True
+```
+
+### C0103 Invalid Name {#C0103}
+
+This error occurs when an invalid name is used as a variable name. This type of 
+error is a sign that the variable names should be modified to be of the form: a 
```

Comment:
It's better to just state what form is allowed. Rewrite the second sentence to say "All variable names should be of the form..."